addOns/pscanrulesAlpha/src/main/resources/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -7,6 +7,11 @@ pscanalpha.examplefile.other=This is for information that doesnt fit in any of t
 pscanalpha.examplefile.soln=A general description of how to solve the problem
 pscanalpha.examplefile.refs=http://zaproxy.blogspot.co.uk/2014/04/hacking-zap-3-passive-scan-rules.html
 
+pscanalpha.featurepolicymissing.name=Feature Policy Header Not Set
+pscanalpha.featurepolicymissing.desc=Feature Policy Header is an added layer of security that helps to restrict from unauthorized access or usage of browser/client features by web resources. This policy ensures the user privacy by limiting or specifying the features of the browsers can be used by the web resources. Feature Policy provides a set of standard HTTP headers that allow website owners to limit which features of browsers can be used by the page such as camera, microphone, location, full screen etc.
+pscanalpha.featurepolicymissing.refs=https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy\nhttps://developers.google.com/web/updates/2018/06/feature-policy\nhttps://scotthelme.co.uk/a-new-security-header-feature-policy/\nhttps://w3c.github.io/webappsec-feature-policy/\nhttps://www.smashingmagazine.com/2018/12/feature-policy/
+pscanalpha.featurepolicymissing.soln=Ensure that your web server, application server, load balancer, etc. is configured to set the Feature-Policy header, to achieve optimal browser support: "Feature-Policy" for Chrome 60+, Firefox 65+ and Opera 47+.
```

Comment:
Remove `to achieve optimal browser...`? Redundant, it's not suggesting other header.